### ISSUE TITLE
How to only output to stdout and not to file

### ISSUE THREAD

**Issue by kootenpv** [brmmm3/fastlogging#7]
Note that I did try `pathName=None`, but it still tries to open it.

And will it still give significant time improvement?

**Comment by brmmm3** [comment]
Are you sure?

If you do this then it still tries to open a file?
```
from fastlogging import LogInit
logger = LogInit(console = True, colors = True)
```

**Comment by kootenpv** [comment]
> Are you sure?
> 
> If you do this then it still tries to open a file?
> 
> ```
> from fastlogging import LogInit
> logger = LogInit(console = True, colors = True)
> ```

yeah

![image](https://user-images.githubusercontent.com/2746227/166220436-3a9cd126-d09d-4b0e-8871-79462224d9d3.png)

**Comment by brmmm3** [comment]
Good catch. I've solved this issue with new version 1.0.0 which you can get from this site. I'll update pypi a bit later.
Thank you for the bug report.
